packages/core/src/utils/modules/encode-module-msg.ts

```diff
@@ -0,0 +1,16 @@
+import { jsonToBinary } from '@abstract-money/core'
+import { ModuleType } from '../../codegen/gql/graphql'
```

Comment:
This module type might be the wrong one? Which one is preferred to use?

Reply:
it's always preferred to use types from the contracts codegen.

Reply:
IN the legacy SDK we used `VariantKeys<ModuleReference>` which we could use from the contracts codegen... thoughts?